<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #b3dbd3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>350</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>70</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white
</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameInputBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordInputBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>70</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white
</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="MoveBuddyTitle">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>140</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;MoveBuddy&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="loginTitle">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="background">
      <property name="geometry">
       <rect>
        <x>-4</x>
        <y>6</y>
        <width>351</width>
        <height>301</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="usernameTitle">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="passwordTitle">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="incorrectLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Username or password incorrect!</string>
      </property>
     </widget>
     <zorder>background</zorder>
     <zorder>loginButton</zorder>
     <zorder>usernameInputBox</zorder>
     <zorder>passwordInputBox</zorder>
     <zorder>exitButton</zorder>
     <zorder>MoveBuddyTitle</zorder>
     <zorder>loginTitle</zorder>
     <zorder>usernameTitle</zorder>
     <zorder>passwordTitle</zorder>
     <zorder>incorrectLabel</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
